foundation/foundation-upgrade/src/test/resources/test-files/v1_8_0/AiopsReferenceJavaMigration/validation/IngestValidated.java

```diff
@@ -0,0 +1,146 @@
+package org.boozallen.aissemble;
+
+/*-
+ * #%L
+ * authzforce::Pipelines::Spark Pipeline
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.core.filestore.AbstractFileStore;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.spark.sql.api.java.UDF1;
+import org.apache.spark.sql.api.java.UDF2;
+import com.boozallen.aissemble.data.encryption.policy.config.EncryptAlgorithm;
+import org.apache.spark.sql.types.DataTypes;
+
+import javax.inject.Inject;
+import java.util.Map;
+import java.util.HashMap;
+import org.boozallen.aissemble.pipeline.PipelineBase;
+import com.boozallen.aissemble.security.client.PolicyDecisionPointClient;
+import com.boozallen.aissemble.security.authorization.policy.PolicyDecision;
+import com.boozallen.aissemble.data.encryption.policy.EncryptionPolicy;
+import com.boozallen.aissemble.data.encryption.policy.EncryptionPolicyManager;
+import com.boozallen.aissemble.security.exception.AissembleSecurityException;
+import com.boozallen.aissemble.security.authorization.models.PDPRequest;
+import com.boozallen.aissemble.data.encryption.AiopsEncrypt;
+import com.boozallen.aissemble.data.encryption.SimpleAesEncrypt;
+import com.boozallen.aissemble.data.encryption.VaultEncrypt;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.functions;
+import org.apache.spark.sql.types.StructType;
+import org.apache.commons.lang.NotImplementedException;
+
+import java.util.stream.Collectors;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.HashSet;
+
+import org.apache.spark.sql.Encoder;
+import org.apache.spark.sql.Encoders;
+
+import static org.apache.spark.sql.functions.col;
+import static org.apache.spark.sql.functions.lit;
+
+import org.aeonbits.owner.KrauseningConfigFactory;
+
+/**
+ * Performs scaffolding synchronous processing for Ingest. Business logic is delegated to the subclass.
+ *
+ * GENERATED CODE - DO NOT MODIFY (add your customizations in Ingest).
+ *
+ * Generated from: templates/data-delivery-spark/synchronous.processor.base.java.vm
+ */
+public abstract class Ingest extends AbstractPipelineStep {
+
```

Comment:
The Class name doesn't match the Java file name.  Should this class be named IngestValidated?

Reply:
Corrected this test file :)